scripts/menu.sh

```diff
@@ -443,6 +443,9 @@ function benchmarking() {
 
 function main() {
   export MENU=true
+  # The following two lines constitute a workaround for a bug where the first dialog call after startup fails before user input is possible.
```

Comment:
Wow, strange... not sure why that would be the case.

Reply:
![image](https://user-images.githubusercontent.com/52489/56344316-f61ff900-6171-11e9-9d74-99bdfb91907c.png)


Btw, `dialog` supports `--sleep` argument